packages/vitest/src/runtime/collect.ts

```diff
@@ -97,6 +105,8 @@ function interpretTaskModes(suite: Suite, namePattern?: string | RegExp, onlyMod
     else if (t.type === 'suite') {
       if (t.mode === 'skip')
         skipAllTasks(t)
+      else if (!interpreted)
+        interpretTaskModes(t, namePattern, onlyMode, isIncluded)
```

Comment:
Shouldn't this be `includeTasks`? Given that you want to always run `interpretTaskModes` looks better to do so here for both cases and remove the need for the `interpreted` boolean

Reply:
I updated my changes